app/Access/LdapService.php

```diff
@@ -82,17 +82,24 @@ public function getUserDetails(string $userName): ?array
         $idAttr = $this->config['id_attribute'];
         $emailAttr = $this->config['email_attribute'];
         $displayNameAttr = $this->config['display_name_attribute'];
+        $firstName = explode(',', $displayNameAttr)[0];
+        $lastName = explode(',', $displayNameAttr)[0];
```

Comment:
@brijm-improwised it should be 1. Also, you should check length before accessing index 1